path_login_test.go

```diff
@@ -212,11 +205,8 @@ func TestLogin(t *testing.T) {
 
 	resp, err = b.HandleRequest(context.Background(), req)
 	if err == nil {
-		t.Fatal("expected error")
-	}
-	var expectedErr error
-	expectedErr = multierror.Append(expectedErr, errwrap.Wrapf("failed to validate JWT: {{err}}", errMismatchedSigningMethod), errwrap.Wrapf("failed to validate JWT: {{err}}", rsa.ErrVerification))
-	if err.Error() != expectedErr.Error() {
+		t.Fatalf("Expected error")
+	} else if err.Error() != "permission denied" {
```

Comment:
Is it possible to check against `logical.ErrPermissionDenied` directly?

Reply:
Good call.